Classes/Query.php

```diff
@@ -871,6 +872,23 @@ public function removeFilterByKey($key)
     }
 
     /**
+     * Removes a filter by the filter value. The value has the following format:
+     *
+     * "fieldname:value"
+     *
+     * @param string $filterString The filter to remove, in the form of field:value
```

Comment:
Why not split this up into two parameters `$key` and `$value`?
Who know if the colon will be changed? Or if at some point the API changes in other ways...